src/support/Default.ts

```diff
@@ -1,9 +0,0 @@
-import 'reflect-metadata';
-import {defaultMetadataStorage} from "./storage";
-import {DefaultValueCallbackMetadata} from "./metadata/MetadataStorage";
-
-export function Default<T>(callback: () => T, condition: (value: T) => boolean = (value) => value === null): (target: Object, propertyKey: string) => void {
```

Comment:
Same here - why is it removed?

Reply:
Default value is literally what you assign to the variable in the entity definition. I thought the conditioning part is probably barely used.

Reply:
Explained further in the PR description.